### ISSUE TITLE
ollama run model error

### ISSUE THREAD

**Issue by DankerMu** [OpenBMB/ollama#20]
### What is the issue?

I had built ollama and it can pull and create model normally,but when i run models, they all give the error:an unknown error was encountered while running the model
![Uploading 1723341239066.png…]()

### OS

Linux

### GPU

Nvidia

### CPU

Intel

### Ollama version

0.0.0 build myself according to the tutor

**Comment by DankerMu** [comment]
![1723341239066](https://github.com/user-attachments/assets/d270c1a9-0bfe-4a18-b1dd-80a28b6258e4)

**Comment by mediawall** [comment]
I have the similar issue with log:
llama_get_logits_ith: invalid logits id 13, reason: no logits

**Comment by Fridayfairy** [comment]
Yes, I also encountered this issue
<img width="1047" alt="image" src="https://github.com/user-attachments/assets/deacd0a4-98ca-44e7-a316-9399a43a959d">
